src/bokeh/models/annotations/geometry.py

```diff
@@ -62,6 +68,10 @@
     "Whisker",
 )
 
+Coordinate = Either(CoordinateLike, Instance(Node))
+
+NaN = float("nan")
```

Comment:
This seems like it should go in a util module (and replace many duplicate instances of the same) but and issue will suffice if you don't want to add that change to this PR

Reply:
Apparently there is not `math.nan` and `math.inf`, so I went ahead and used that and normalized all similar usage.

Reply:
I moved `Coordinate` to `bokeh.models.common.properties`. Can't use `bokeh.model.util` due to circular imports.